ingest/muxer.go

```diff
@@ -1348,14 +1316,24 @@ inputLoop:
 
 			e := ee.(*entry.Entry)
 
-			ttag, ok = nc.tt.Translate(e.Tag)
-			if !ok {
+			if ttag, err = nc.translateTag(e.Tag); err != nil {
 				// If the ingest muxer has no idea what this tag is, drop it and notify
 				if name, ok := im.LookupTag(e.Tag); !ok {
```

Comment:
The "does the muxer even know about this?" check should occur before we attempt to negotiate

Reply:
the muxer lookup is a map lookup where the translate is an order 1 vector lookup, realistically, you would have to be using the muxer wrong to get here, so I don't think taking the performance hit is worth it here when you have to be going out of your way to even get here.